LAB4JS/tasks.js

```diff
@@ -0,0 +1,154 @@
+function task1() {
+    let fruits = ['яблуко', 'банан', 'вишня', 'груша'];
+    fruits.pop();
+    document.getElementById('task1').innerText += '1.1: ' + fruits.join(', ') + '\n';
+
+    fruits.unshift('ананас');
+    document.getElementById('task1').innerText += '1.2: ' + fruits.join(', ') + '\n';
+
+    fruits.sort().reverse();
+    document.getElementById('task1').innerText += '1.3: ' + fruits.join(', ') + '\n';
+
+    document.getElementById('task1').innerText += '1.4: Індекс яблука: ' + fruits.indexOf('яблуко') + '\n';
+}
+
+function task2() {
+    let colors = ['червоний', 'синій', 'зелений', 'жовтий', 'темно-синій'];
+    let longest = colors.reduce(function(a, b) {
+        return a.length > b.length ? a : b;
+    });
+    let shortest = colors.reduce(function(a, b) {
+        return a.length < b.length ? a : b;
+    });
+
+    document.getElementById('task2').innerText += '2.1: Довгий: ' + longest + ' Короткий: ' + shortest + '\n';
```

Comment:
Display the results of functions in the browser console with the task number for easy verification